charts/actions-runner-controller/templates/deployment.yaml

```
@@ -65,6 +65,9 @@ spec:
         {{- if .Values.runnerGithubURL  }}
         - "--runner-github-url={{ .Values.runnerGithubURL }}"
         {{- end }}
+        {{- if .Values.useCustomRunnersRBAC  }}
```

```suggestion
        {{- if .Values.useRunnerStatusUpdateHookEphemeralRole  }}
```

After reading all this, perhaps this might better reflect (1)how log the custom RBAC resources persist(=as long as the runner pod exists) and (2)the purpose of the custom RBAC resources(=to enable runner status update hook in runner pods)?